pom.xml

```diff
@@ -3371,5 +3371,25 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>partial-build</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.lesfurets</groupId>
+                        <artifactId>partial-build-plugin</artifactId>
+                        <version>2.6</version>
+                        <extensions>true</extensions>
```

Comment:
should be extension without s, no?

Reply:
maven fails if not.

I inspected the XSD, and it's `extensions`. Probably a typo on the project README.